training/architecture/DATA_TERMS.md

```diff
@@ -0,0 +1,57 @@
+Common Data Terms
+===========================
+
+This training document breaks down common terms we use to describe our data, and the relationships between different data types.
+
+## Pre-reading
+None.
+
+## Materials
+
+### What do we publish? Datasets, editions and versions
+
```

Comment:
It might be worth defining what data is. There are a lot of misconceptions floating around and everyone has a slightly different understanding as it can be used as quite a generic term.

Previously 'data' and 'information' were taught to me as:
- Data on its own means nothing and is just (a single or series of) numbers, letters or whatever e.g.  '60' 'Rodger', '2021'
- Information is labeled data 'Age: 60', 'Name: Rodger', 'Current year: 2021'
There are probably better definitions out there than this mind